contracts/sol6/wrappers/KyberRateHelper.sol

```diff
@@ -207,4 +232,112 @@ contract KyberRateHelper is IKyberRateHelper, WithdrawableNoModifiers, Utils5 {
             );
         }
     }
+
+    function getSpreadInfo(IERC20 token, uint256 optionalAmountWei)
+        public
+        view
+        returns (bytes32[] memory reserves, uint256[] memory spreads)
+    {
+        (
+            bytes32[] memory buyReserves,
+            uint256[] memory buyRates,
+            bytes32[] memory sellReserves,
+            uint256[] memory sellRates
+        ) = getReservesRates(token, optionalAmountWei);
+
+        uint256[] memory revertReserveIndex = new uint256[](buyReserves.length);
+        uint256 validReserve = 0;
+        for (uint256 i = 0; i < buyRates.length; i++) {
+            if (buyRates[i] == 0) {
+                continue;
+            }
+
+            for (uint256 j = 0; j < sellRates.length; j++) {
+                if (sellReserves[j] == buyReserves[i]) {
+                    if (sellRates[j] == 0) break;
+                    revertReserveIndex[i] = j;
+                    validReserve++;
+                    break;
+                }
+            }
+        }
+        reserves = new bytes32[](validReserve);
+        spreads = new uint256[](validReserve);
+        uint256 reserveIndex;
+        for (uint256 i = 0; i < buyRates.length; i++) {
+            if (buyRates[i] == 0) {
+                continue;
+            }
+            reserves[reserveIndex] = buyReserves[i];
+            spreads[reserveIndex] = calcSpreadInBps(buyRates[i], sellRates[revertReserveIndex[i]]);
+            reserveIndex++;
+        }
+    }
+
+    function getSlippageRateInfo(
+        IERC20 token,
+        uint256 optinalBaseAmount,
+        uint256 optinalSlippageAmount
+    )
+        public
+        view
+        returns (
+            bytes32[] memory buyReserves,
+            int256[] memory buySlippageRateBps,
+            bytes32[] memory sellReserves,
+            int256[] memory sellSlippageRateBps
+        )
+    {
+        uint256 baseAmount = optinalBaseAmount > 0 ? optinalBaseAmount : 0.01 ether;
```

Comment:
0.01 works?
did u test in remix?

anyway let's have default values as constant on top of contract

Reply:
> 0.01 works?

`0.01` ether works
> anyway let's have default values as constant on top of contract

Ok, let me update this